docker/dendrite.Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN tar xf go.tar.gz -C /goroot --strip-components=1
 ENV GOROOT=/goroot
 ENV GOPATH=/gopath
 ENV PATH="/goroot/bin:${PATH}"
+ENV SYTEST_DEFAULT_BRANCH dendrite
```

Comment:
a comment here would be good. What does this do, and why is it useful? (I shouldn't have to go digging around in a bunch of other files to figure it out)